sections/ai_rules.tex

```diff
@@ -5,7 +5,7 @@
 
 \begin{multicols}{2}
 
-\subsection*{AI Hero Turn}
+\subheader{AI Hero Turn}
```

Comment:
This text is strangely glued to the header now. Why is that happening? The gap between the text and the header should be equal to the one between the header and the top edge of the page.

Reply:
What about this?

with 2 subheaders it doesn't conflict with main header anymore as much

<img width="2482" height="1754" alt="en-44" src="https://github.com/user-attachments/assets/1121e10c-fc47-4572-a854-ac858737aa59" />

Reply:
Better yeah. Let's hope it doesn't get too tight for translations, as many have struggled with this.